src/main/java/org/terasology/worldlyTooltip/ui/WorldlyTooltip.java

```diff
@@ -111,8 +122,7 @@ public String get() {
                         @Override
                         public List<TooltipLine> get() {
                             if (cameraTargetSystem.isTargetAvailable()) {
-                                EntityRef targetEntity = blockEntityRegistry.getEntityAt(cameraTargetSystem.getTargetBlockPosition());
-
+                                EntityRef targetEntity = cameraTargetSystem.getTarget();
```

Comment:
This works ok for blocks too, right?  If so,  I wonder why I chose to use the block entity registry... Good job finding a more generic solution.